src/app/client/src/app/app.component.ts

```diff
@@ -195,10 +196,12 @@ export class AppComponent implements OnInit, OnDestroy {
           if (!deviceProfile.userDeclaredLocation ||
             !(this.usersProfile && this.usersProfile.userLocations && this.usersProfile.userLocations.length >= 1)) {
             this.isLocationConfirmed = false;
+            this.popupControlService.changePopupStatus('isLocationConfirmed', false);
```

Comment:
the if and else are doing the same call why do you want to call it 2times

Reply:
Sir That If-Else is already been written. I have No idea about it.